protos/getresource.proto

```diff
@@ -3,9 +3,8 @@ syntax = "proto3";
 package getresource;
 
 message GetResourceRequest {
-    string KbcName = 1;
-    string KbsUri = 2;
-    string ResourceDescription = 3;
+    string ResourceUri = 1;
```

Comment:
I changed the Proto here, as kbsUri is part of the resourceUri. KbcName now still matters and I think we can open an issuer later to talk about https://github.com/confidential-containers/attestation-agent/issues/99 on both image-rs side and kata-agent side.